.github/workflows/gradle.yml

```diff
@@ -5,8 +5,14 @@ on: [push]
 jobs:
   build:
 
-    runs-on: oracle/graalvm-ce
+    name: Run Tests
+    runs-on: ubuntu-18.04
 
     steps:
-    - name: Run Tests with Gradle
-      run: ./gradlew clean functionalTest
+      - uses: actions/checkout@latest
```

Comment:
this needs to be `@v1` instead of `@latest`
see https://github.com/christian-draeger/kotlin-cli-example/blob/master/.github/workflows/gradle.yml
for a working example